Shop.DAL/Common/RequestParams/CategoryParameters.cs

```diff
@@ -0,0 +1,4 @@
+namespace Shop.DAL.Common.RequestParams
+{
+   public record CategoryParameters : RequestParameters;
```

Comment:
what are these models?

Reply:
These models represent collection filtering, like paging, searching and filtering. Since all these operations perform on database side, it should be passed to repository. Request it these context means not HTTP request, but request for data from repository